src/richie/apps/core/templatetags/feature_flags.py

```diff
@@ -0,0 +1,15 @@
+"""Custom template tags related to Joanie."""
+from django import template
+from django.conf import settings
+
+register = template.Library()
+
+
+@register.simple_tag()
+def features(flag):
```

Comment:
I should catch it sooner, but by convention we try to write method whose return a boolean as a question (`is_enabled`, `has_course_runs`, ...)

What do you think to rename this template tag as follow ?
```suggestion
def is_feature_enabled(flag):
```